lib/encodings/base64.fz

```diff
@@ -0,0 +1,155 @@
+# Base64 encoding and decoding as defined in RFC 4648
+public base64 is
+
+  # allows redefinition for base64url
+  module get_alphabet => "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/".as_codepoints.as_array
+  module get_char_62 u8 => 43 # +
```

Comment:
get_char_62 and get_char_63 seems same as just calling get_alphabet[62], get_alphabet[63]

Reply:
you're right, its also less error-prone, having to change things in one place only